src/elements/Flex/Flex.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import { numberLikeProp } from '../../util/helpers';
+import Common from '../Common/Common';
+
+const Flex = ({ children, max, fixed, centerY, ...other }) => {
+	const combinedClasses = classNames(
+			'ui-flex-item',
+			{
+				'center-y': centerY
+			},
+			other.classes
+		),
+		style = {};
+
+	if(max) style.maxWidth = Number(max);
+	if(fixed) {
+		style.flexGrow = 0;
+		style.flexShrink = 0;
+		style.flexBasis = Number(fixed);
+	}
+
+	/**
+	 * Render the grid items (`renderGridItems` defined below)
```

Comment:
The latter part of this comment isn't relevant (and actually needs to be changed on the Grid.js too)

Reply:
copypasta, fixing